util/cron/test-perf.cray-cs.ofi.bash

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Run performance tests with comm=ofi on a cray-cs system.
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs'
+
+source $CWD/common-perf.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.cray-cs.ofi"
+
+module load gcc
+module load python/2.7.6
+module load pmix
+
+export CHPL_HOST_PLATFORM=cray-cs
+export CHPL_COMM=ofi
+export CHPL_COMM_OFI_OOB=slurm-pmi2
+export SLURM_MPI_TYPE=pmix
+export CHPL_RT_MAX_HEAP_SIZE=83G
```

Comment:
Does `CHPL_RT_MAX_HEAP_SIZE` need to be set?

Reply:
Yes, the `verbs;ofi_rxm` provider we use on CS systems doesn't have full scalable memory registration, and without scalable MR we currently have to specify the heap size.  (The situation is much like the early days of comm=ugni on XC systems.)